app/assets/stylesheets/profile.scss

```diff
@@ -113,19 +115,23 @@
   }
 
   #profile {
-    padding: 20px;
     margin-bottom: 35px;
+    border-radius: 20px 20px 0px 0px;
 
     #profile_photo {
       margin-top: 10px;
-      padding-bottom: 10px;
       text-align: center;
     }
 
+    #profile_photo img {
+      border-radius: 20px 20px 0px 0px;
+      border: 1px solid #c1c1c1;
```

Comment:
Color literals like `#c1c1c1` should only be used in variable declarations; they should be referred to via variable everywhere else.